src/art/art.c

```diff
@@ -40,6 +40,10 @@ namespace roaring {
 namespace internal {
 #endif
 
+void art_iterator_init(art_iterator_t *iterator) {
```

Comment:
Nit/style: I suggest moving this to line 123 or somewhere around there, so all typedefs are at the top before utility functions.